### PR TITLE
Today is sunday

### PR DESCRIPTION
**Pull request by Hemachandra-423**
Please rveiw

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <div class="container-fluid main">
 	<div class="row-fluid">
 		<div class="span10 offset1">
-			<h1> TS-6 Happy Dasara</h1>
+			<h1> Today is  saturday</h1>
 			h1> TS-6 Wishing you all the best for the CJE</h1>
```
Comment (mohansgithub) [inline src/main/webapp/WEB-INF/views/home.jsp]: Add india is my country

**Review by mohansgithub — CHANGES_REQUESTED** [review]
please fix

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 2]
```diff
@@ -7,7 +7,7 @@
 <div class="container-fluid main">
 	<div class="row-fluid">
 		<div class="span10 offset1">
-			<h1> TS-6 Happy Dasara</h1>
+			<h1> Today is  saturday</h1>
```
Comment (selangovan400) [inline src/main/webapp/WEB-INF/views/home.jsp:10]: its good.

**Review by selangovan400 — CHANGES_REQUESTED** [review]
Please fix.

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 1 above)
Comment (Hemachandra-423) [reply src/main/webapp/WEB-INF/views/home.jsp]: Its fixed

Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp]: Good

**Review by mohansgithub — APPROVED** [review]
Appproved from mohanraj

**Review by selangovan400 — APPROVED** [review]
Approved.

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 3]
```diff
@@ -7,8 +7,9 @@
 <div class="container-fluid main">
 	<div class="row-fluid">
 		<div class="span10 offset1">
-			<h1> TS-6 Happy Dasara</h1>
-			h1> TS-6 Wishing you all the best for the CJE</h1>
+			<h1> Today is  saturday</h1>
```
Comment (maheshdevops44) [inline src/main/webapp/WEB-INF/views/home.jsp:10]: approved

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 4]
```diff
@@ -7,8 +7,9 @@
...
+			<h1> TS-6 Wishing you all the best for the CJE</h1>
+			<h1> India is my country</h1>
```
Comment (maheshdevops44) [inline src/main/webapp/WEB-INF/views/home.jsp:12]: approved

**Review by maheshdevops44 — COMMENTED** [review]
approve

**Pull request merged**